core/wren/src/wren/docs_content/refs/mdl.md

```
@@ -0,0 +1,381 @@
+  ORDER BY 2 DESC
+  LIMIT 100
+properties:
+  description: "Top customers by lifetime value"
+```
+
+| Field | Type | Required | Description |
+|---|---|---|---|
+| `name` | string | yes | Unique view name. |
+| `statement` | string | yes | Complete SQL SELECT. May reference other models or views. |
+| `dialect` | string | no | SQL dialect (metadata only — engine parses with its generic parser). Requires `schema_version: 3`. |
+| `properties` | no | Arbitrary metadata. |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 4; Actual: 3; Too few cells, row will be missing data

(MD056, table-column-count)